swiggyclone/service/views.py

```diff
@@ -22,45 +22,74 @@
 
 class RestaurentViewSet(viewsets.ModelViewSet):
     queryset = Restaurent.objects.all().prefetch_related('branches')
-    serializer_class = RestaurentSerializers
+    serializer_class = RestaurentSerializer
 
 
 class BranchViewSet(viewsets.ModelViewSet):
     queryset = Branch.objects.all().prefetch_related('foodItems')
```

Comment:
you have defined `queryset` variable as well as `get_queryset()` method. have you checked both is required or only one is enough in your case?